package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "react-accessible-dropdown-menu-hook",
-	"version": "2.2.2",
+	"version": "2.3.2",
```

Bumping the minor version resets the patch version:

```suggestion
	"version": "2.3.0",
```